src/main/java/coydir/storage/JsonAdaptedRating.java

```diff
@@ -0,0 +1,46 @@
+package coydir.storage;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import coydir.commons.exceptions.IllegalValueException;
+import coydir.model.person.Rating;
+import coydir.model.tag.Tag;
+
+/**
+ * Jackson-friendly version of {@link Rating}.
+ */
+public class JsonAdaptedRating {
+    private final String value;
+    private final String timestamp;
+
+    /**
+     * Constructs a {@code JsonAdaptedLeave} with the given {@code startDate} and {@code endDate}.
+     */
+    @JsonCreator
+    public JsonAdaptedRating(@JsonProperty("rating") String value, @JsonProperty("timestamp") String timestamp) {
+        this.value = value;
+        this.timestamp = timestamp;
+    }
+
+    /**
+     * Converts a given {@code Leave} into this class for Jackson use.
+     */
+    public JsonAdaptedRating(Rating source) {
+        this.value = String.valueOf(source.value);
+        this.timestamp = String.valueOf(source.timestamp);
```

Comment:
Prob need to change the timestamp to String Formatted if not it will store in the JSON in the wrong format which will screw up when u restart the application.

Can refer to `JsonAdaptedLeave` 
```suggestion
       this.timestamp = source.timestamp.format(DateTimeFormatter.ofPattern("dd-MM-yyyy"));
```